package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@f1v/zati",
-  "version": "0.0.16",
+  "version": "0.0.0-development",
```

Comment:
`semantic-release` updates the `package.json` version right before publishing and it will be accurate in the package. But they recommend doing this here to avoid inconsistencies

https://semantic-release.gitbook.io/semantic-release/support/faq#why-is-the-package-jsons-version-not-updated-in-my-repository